README.md

```diff
@@ -734,7 +734,9 @@ sops-nix also provides a home-manager module.
 This module provides a subset of features provided by the system-wide sops-nix since features like the creation of the ramfs and changing the owner of the secrets are not available for non-root users.
 
 Instead of running as an activation script, sops-nix runs as a systemd user service called `sops-nix.service`.
-And instead of decrypting to `/run/secrets`, the secrets are decrypted to `$XDG_RUNTIME_DIR/secrets` that is located on a tmpfs or similar non-persistent filesystem.
+And instead of decrypting to `/run/secrets`, the secrets are stored decrypted to `$XDG_RUNTIME_DIR/secrets` that is located on a tmpfs or similar non-persistent filesystem. Additionally secrets are symlinked to the user home-directory in the `.secrets`-directory which is used as reference 
```

Comment:
Sorry need to update this 

1. It's `ls -l $XDG_RUNTIME_DIR/secrets.d/` and there are folders like e.g. "1" which contains the actual secrets-files.

2. this is a left-over from my first proposal and changed later:
Correct is `~/.config/sops-nix/secrets` - will PR soonTM

will PR soonTM (otherwise if you want/can feel free) 🙏🏻

Reply:
Already happend https://github.com/Mic92/sops-nix/pull/550